### PR TITLE
Augment topic::version 3 transaction relay stub

### PR DESCRIPTION
**Pull request by paulkania**
Got tangled up in git. 3rd and hopefully final version, 'building' off [#1004](https://github.com/bitcoinops/bitcoinops.github.io/pull/1004) , and [#1001](https://github.com/bitcoinops/bitcoinops.github.io/pull/1001)

### CODE REVIEW

**Review by Xekyo — APPROVED** [review]
Looks good to me. I have the impression that it should cleanly squash-merge into a single commit on top of our commits.

For the future, you don’t need to merge `master` into your branch, but if you want to make a clean history, you may want to use the latest commit in `master` as your starting point, or reset it to build on top of master calling `git rebase [bitcoinops|origin|upstream] master`  (where the second term is the name that you locally have for the main bitcoinops repository).

**Comment by Xekyo** [comment]
I've rebased on `bitcoinops/master` and squashed all the commits

**Review by LarryRuane — COMMENTED** [review]
A very basic fact -- maybe too basic to mention -- is that a V3 transaction will be accepted by nodes running pre-V3 software, both into their mempool (and relayed) and in blocks. So this isn't a hardfork, and isn't even a softfork, because it affects only _policy_, not _consensus_. So no activation mechanism is needed.

Again, these points may be too basic to mention but are just some things I had to make sure were clear in my mind when I first read about this proposal.

Either way, ACK, this is a good summary!

**Comment by paulkania** [comment]
@Xekyo  - thanks! I'll reference this code-snippet for my next submission, hopefully soon [side-channels stub].  
@LarryRuane  -> i think adding the fact that v3txs are only a change in policy is a good idea, and I'd be happy to edit the article..hopefully without having to make another PR :) ... I'll wait on other opinions here before i rework the page though. Maybe Xekyo wants to merge, i'm  fine with that too.

File: _topics/en/version-3-transaction-relay.md [hunk 1]
```diff
@@ -46,20 +53,19 @@ see_also:
 
   - title: Anchor outputs
     link: topic anchor outputs
+---
+V3 transaction relay is a superset of standard transaction policy.
+That is, V3 transactions follow all rules for standard transactions
+(e.g. minimum and max transaction weight) while also adding some
```
Comment (bitschmidty) [inline _topics/en/version-3-transaction-relay.md:59]:
```suggestion
(e.g. minimum and maximum transaction weights) while also adding some
```

File: _topics/en/version-3-transaction-relay.md [hunk 2]
```diff
@@ -46,20 +53,19 @@ see_also:
...
+additional rules designed to allow [transaction replacement][topic rbf]
+while precluding transaction-pinning attacks. V3 transactions also
+require minor changes to the [Package RBF][topic package relay] policy in order to maintain
```
Comment (bitschmidty) [inline _topics/en/version-3-transaction-relay.md:62]:
```suggestion
require minor changes to the [package RBF][topic package relay] policy in order to maintain
```

File: _topics/en/version-3-transaction-relay.md [hunk 3]
```diff
@@ -46,20 +53,19 @@ see_also:
 
   - title: Anchor outputs
     link: topic anchor outputs
+---
+V3 transaction relay is a superset of standard transaction policy.
+That is, V3 transactions follow all rules for standard transactions
```
Comment (bitschmidty) [inline _topics/en/version-3-transaction-relay.md:58]:
```suggestion
That is, v3 transactions follow all rules for standard transactions
```

File: _topics/en/version-3-transaction-relay.md [hunk 4]
```diff
@@ -46,20 +53,19 @@ see_also:
...
+(e.g. minimum and max transaction weight) while also adding some
+additional rules designed to allow [transaction replacement][topic rbf]
+while precluding transaction-pinning attacks. V3 transactions also
```
Comment (bitschmidty) [inline _topics/en/version-3-transaction-relay.md:61]:
```suggestion
while precluding transaction-pinning attacks. v3 transactions also
```

**Review by bitschmidty — COMMENTED** [review]
@paulkania Thanks for this PR and helping our topics section!

Ive left a few suggestions.

**Comment by Xekyo** [comment]
Ugh. Now I did it wrong myself. I merged this PR in https://github.com/bitcoinops/bitcoinops.github.io/commit/a5fd0faf53a37776f3cb21b8cd59c9e5910f1a39 to `bitcoinops/master`.

**Pull request closed without merging**